File: 문제풀이/프로그래머스/코딩 기초 트레이닝/Level 0. 문자열 겹쳐쓰기.js
function solution(my_string, overwrite_string, s) {
  return my_string.slice(0, s) + overwrite_string + my_string.slice(s + overwrite_string.length);
}

console.log(solution('He11oWor1d', 'lloWorl', 2));
console.log(solution('Program29b8UYP', 'merS123', 7));

File: 문제풀이/프로그래머스/코딩 기초 트레이닝/Level 0. 문자열 겹쳐쓰기.py
def solution(my_string: str, overwrite_string: str, s: int) -> str:
    return my_string[:s] + overwrite_string + my_string[s + len(overwrite_string) :]


print(solution("He11oWor1d", "lloWorl", 2))
print(solution("Program29b8UYP", "merS123", 7))
